kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KafkaSteps.java

```diff
@@ -38,6 +47,36 @@ public static void createTopic(String clusterName, String topicName, String name
                 KafkaTopicTemplates.defaultTopic(namespace, clusterName, topicName, partitions, replicas, minIsr).build());
     }
 
+    /**
+     * Create topic using test clients.
+     *
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param partitions the partitions
+     * @param replicas the replicas
+     */
+    public static void createTopicTestClient(String deployNamespace, String topicName, String bootstrap, int partitions, int replicas) {
```

Comment:
Thinking about API/naming.   Why does someone who is calling the API need to know that the topic will be created by the testclient?   Surely - that's an implementation concern.   So I'd advocate for 'createTopic' as the method name.

Reply:
done. Thanks!